besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogHelper.java

```diff
@@ -97,16 +97,15 @@ private static void processTrieLogBatches(
       final String batchFileNameBase) {
 
     for (long batchNumber = 1; batchNumber <= numberOfBatches; batchNumber++) {
-
+      final String batchFileName = batchFileNameBase + "-" + batchNumber;
```

Comment:
this might make more sense to have first/last block numbers included in the filename.  Otherwise it won't be clear what is actually in the files after an export

Reply:
The batch filenames aren't used as part of import/export subcommands the filename is taken from command line args instead. This is only used for the prune subcommand